server/routes/settings/index.ts

```diff
@@ -252,11 +255,59 @@ settingsRoutes.get('/jellyfin', (_req, res) => {
   res.status(200).json(settings.jellyfin);
 });
 
-settingsRoutes.post('/jellyfin', (req, res) => {
+settingsRoutes.post('/jellyfin', async (req, res, next) => {
```

Comment:
I don't really understand what you're doing here. What is the point of all this new code?

Reply:
This is relevant to how the new abstracted jellyfin hostname settings are saved. Previously when it was being saves, it won't do a check. It just saves. Even if it's wrong.

Now it will check if the url is reachable before saving. This is consistent with how plex settings are saved if you scroll up a little

Reply:
Ok, thanks for the clarification. Make sense.